backend/src/main/java/ca/bc/gov/hlth/hnweb/controller/GroupMemberController.java

```diff
@@ -121,7 +127,11 @@ public ResponseEntity<UpdateNumberAndDeptResponse> updateNumberAndDept(@Valid @R
 
 			// Handle the group member/employee number
 			UpdateNumberAndDeptResponse empNumberResponse = new UpdateNumberAndDeptResponse();
-			if (StringUtils.isNotBlank(updateNumberAndDeptRequest.getGroupMemberNumber())) {
+			String groupMemberNumber = updateNumberAndDeptRequest.getGroupMemberNumber();
+			if (StringUtils.isNotBlank(groupMemberNumber)) {
+				if(groupMemberNumber.length() >= 6 && groupMemberNumber.substring(0,6).contentEquals(SIX_DASHES)) {
```

Comment:
Missing space after opening parenthesis. Also, I think this needs to be an exact match on the dashes. The current code would accept '------ABC' from an API call.